app/services/get_from_doit.rb

```diff
@@ -0,0 +1,30 @@
+class GetFromDoit
+
+	def self.run
+		host = 'https://api.do-it.org'
+    href = "/v1/opportunities\?lat\=51.5978\&lng\=-0.3370\&miles\=0.5 "
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)